ui/app/AppLayouts/Profile/views/keycard/DetailsView.qml

```diff
@@ -0,0 +1,189 @@
+        target: root.keycardStore.keycardModule
+
+        onKeycardProfileChanged: {
+            if (keycardDetails.keyPairType === Constants.keycard.keyPairType.profile) {
+                d.resetKeycardDetails()
+            }
+        }
+
+        onKeycardDetailsChanged: {
+            if (kcUid === root.keycardUid) {
+                d.resetKeycardDetails()
+            }
+        }
+    }
+
+    KeycardItem {
+        id: keycardDetails
+        Layout.fillWidth: true
+    }
+
+    Item {
+        Layout.fillWidth: true
+        Layout.preferredHeight: Style.current.halfPadding
+    }
+
+    StatusSectionHeadline {
+        Layout.fillWidth: true
+        Layout.leftMargin: Style.current.padding
+        Layout.rightMargin: Style.current.padding
+        text: qsTr("Configure your Keycard")
+    }
+
+    StatusListItem {
+        Layout.fillWidth: true
+        title: qsTr("Rename Keycard")
+        components: [
+            StatusIcon {
+                icon: "chevron-down"
+                rotation: 270
+                color: Theme.palette.baseColor1
+            }
+        ]
+        onClicked: {
+            root.keycardStore.runRenameKeycardPopup()
+        }
+    }
+
+    StatusListItem {
+        Layout.fillWidth: true
+        title: qsTr("Change PIN")
+        components: [
+            StatusIcon {
+                icon: "chevron-down"
+                rotation: 270
+                color: Theme.palette.baseColor1
+            }
+        ]
+        onClicked: {
+            root.keycardStore.runChangePinPopup()
+        }
+    }
+
+    StatusListItem {
+        Layout.fillWidth: true
+        title: qsTr("Create a backup copy of this Keycard")
+        components: [
+            StatusIcon {
+                icon: "chevron-down"
+                rotation: 270
+                color: Theme.palette.baseColor1
+            }
+        ]
+        onClicked: {
+            root.keycardStore.runCreateBackupCopyOfAKeycardPopup()
+        }
+    }
+
+    StatusListItem {
+        visible: keycardDetails.keycardLocked
+        Layout.fillWidth: true
+        title: qsTr("Unlock Keycard")
+        components: [
+            StatusBadge {
+                value: 1 //always set to 1 if keycard is locked
+                border.width: 4
+                border.color: Theme.palette.dangerColor1
+                color: Theme.palette.dangerColor1
+            },
+            StatusIcon {
+                icon: "chevron-down"
+                rotation: 270
+                color: Theme.palette.baseColor1
+            }
+        ]
+        onClicked: {
+            root.keycardStore.runUnlockKeycardPopupForKeycardWithUid(root.keycardUid)
+        }
+    }
+
+    StatusListItem {
+        Layout.fillWidth: true
+        title: qsTr("Advanced")
+        statusListItemTitle.color: Style.current.secondaryText
+        components: [
+            StatusIcon {
+                icon: "chevron-down"
+                rotation: d.collapsed? 0 : 180
```

Comment:
Afaik we have proper chevron icons (all directions), no need for the rotation (also above)